CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Documents changes that result in:
 ## [0.30.0](https://github.com/raiden-network/raiden-contracts/releases/tag/v0.30.0) - 2019-08-15
 
 - [#1150](https://github.com/raiden-network/raiden-contracts/pull/1150) Use different message ID for chnanelClose() and updateNonClosingBalanceProof()
+- [#1163](https://github.com/raiden-network/raiden-contracts/pull/1163) MonitoringService.monitor() no longer works for service providers that are not registered in ServiceRegistry
```

Comment:
This should move out of the `0.30.0` section.

Reply:
Thank you for spotting this.